kernel/enter.c

```diff
@@ -170,6 +169,16 @@ static int _copy_host_etc_files()
     return ret;
 }
 
+static int _copy_host_etc_files()
+{
+    int ret = 0;
+    ECHECK(_copy_host_etc_file("/etc/resolv.conf"));
```

Comment:
@bodzhang @paulcallen @mikbras @jxyang 

The default value for `timeout` option is 5. The value is capped to 30.
See https://man7.org/linux/man-pages/man5/resolver.5.html

Given that we've seen timeout (EAI_AGAIN) errors in CI, I'm led to believe that dns resolution in Mystikos is slower than normal - could be due to SGX overhead, CI network issues etc.

One way to ensure minimize timeout errors is to append the line `options timeout:30` to the resolv.conf file.
What do you think? 

Many developers may not be aware what to do when dns resolution timeouts happen. So, automatically doing it for them could lead to better developer experience.

Reply:
@paulcallen pointed out that the default was `5 seconds` which is already sufficiently large. Therefore, there is no need to increase the timeout to 30 seconds.